k2/csrc/tensor_test.cu

```diff
@@ -4,6 +4,7 @@
  *
  * @copyright
  * Copyright (c)  2020  Xiaomi Corporation (authors: Haowen Qiu)
+ *                      Fangjun Kuang (csukuangfj@gmail.com)
```

Comment:
just to make sure you do this (not adding your company information here) intentionally :)

Reply:
Fixed.